### ISSUE TITLE
Using 'and' joiner instead of 'or' for multiple facets

### ISSUE THREAD

**Issue by adehner** [Daniel-KM/Omeka-S-module-AdvancedSearch#10]
When selecting multiple facet values for a single property, the search appears to be joined with OR instead of AND. Is this expected behavior with the internal querier? Example below, to help explain my question.

**When I select the value "Labor & Class", I get all items (count of 6) with the "Type of Social Identifier = "Labor & Class.**
<img width="1173" alt="Screenshot 2023-11-13 at 2 10 56 PM" src="https://github.com/Daniel-KM/Omeka-S-module-AdvancedSearch/assets/6174191/7cf54c9c-f456-4b22-8100-9752433837c8">

**When I add the facet value "Family Structures", I would like to see the items that have both values. Instead I see items (count of 15) that one value or the other.**
<img width="1171" alt="Screenshot 2023-11-13 at 2 11 24 PM" src="https://github.com/Daniel-KM/Omeka-S-module-AdvancedSearch/assets/6174191/d15aa116-a1b3-4c90-a710-5d602b2d3417">

**Comment by Daniel-KM** [comment]
Yes, some want a "and" and some want "or." So maybe an option should be added to the config in a future version.

**Comment by adehner** [comment]
If I wanted to modify the operator, can you point me to the change(s) I would need to make?

**Comment by jthiels** [comment]
In a pool search query:
`property[0][joiner]=and&property[0][property][0]=81&property[0][type]=eq&property[0][text]=Published&property[1][joiner]=and&property[1][property][0]=9&property[1][type]=eq&property[1][text]=Article
`
the browse page is taking the query as Article and Published OR property[0] (which is the Title, so if there is more than one entry under the Title, then it is giving us all published Articles.  Is there any way to make this AND instead of OR?

**Comment by Numerizen** [comment]
The default behaviour still seems to be 'OR' for full text search.

is there a way to tell Omeka to use the 'AND' operator by default ?

**Comment by Daniel-KM** [comment]
Implemented in 3.4.64.

**Comment by adehner** [comment]
When will 3.4.64 be released?
